source/entities/ISelectable.hx

```diff
@@ -7,39 +7,46 @@ import flixel.util.FlxSpriteUtil.LineStyle;
 import flixel.group.FlxGroup;
 import flixel.util.FlxColor;
 import flixel.FlxSprite;
+import render.WorldConstants;
 
 using flixel.util.FlxSpriteUtil;
 
 class ISelectable extends FlxSpriteGroup {
```

Comment:
Does it make sense to use this for the tower ranges? The other build/upgrade options will likely appear when the character is standing near the tower, so we don't have to handle any explicit selection for that aspect.